### PR TITLE
Add code of conduct page

### PR DESCRIPTION
**Pull request by MartinSchoeler**

### CODE REVIEW

File: legal/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+title: Code of Conduct
+class: legal
+permalink: /code-of-conduct
+type: code-of-conduct
+theme: light
+redirect_from:
+  - /code-of-conduct/
+---
+
+_Effective date: December 18, 2017_
```
Comment (MartinSchoeler) [inline legal/code-of-conduct.md]: Is this date right?

File: legal/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Rocket.Chat provides numerous spaces for our community to interact and experience our product. Among these are:
+[Open.rocket.chat](https://open.rocket.chat/), an open chat community for bringing people together and to showcase Rocket.Chat´s community edition. In addition, new features are tested on the server on a regular basis. 
+[github.com/RocketChat](http://github.com/RocketChat), our source code repositories where we discuss issues and plan new features
+[Forums.rocket.chat](https://forums.rocket.chat/), our webforum to announce new features and provide ways for direct interaction with our developers
+
+## Introduction
+
+The main objective of the rocket.chat community spaces is bringing people together to discuss and experience Rocket.Chat - our awesome communication platform. This Code of Conduct is mainly written in mind for Open.rocket.chat, but in its spirit will be applied to other Rocket.Chat community spaces as well.
+
+When new visitors enter a channel, you should acknowledge their arrival especially if you're engaged in social chit-chat / passing time. Failure to do so is likely to result in them leaving the channel feeling unsupported.
```
Comment (geekgonecrazy) [inline legal/code-of-conduct.md]:
```suggestion
When new visitors enter a channel, you should acknowledge their arrival especially if you're engaged in social chit-chat / passing time. Failure to do so is likely to result in them leaving the channel feeling unwelcome.
```

Maybe?

File: legal/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+You should try to minimize the chances of conflict by being open-minded and accepting of others. Understand that we each have a unique set of values, and we are all entitled to our own opinions.
+
+Feel free to share your views on someone else's problem as an outsider, but avoid using excessive force, and don't expect your thoughts and ideas to be accepted all of the time.
+
+We're mainly an English speaking community but welcome all nationalities. If you find yourself speaking primarily in a different language in an open channel or discussion, please PM that person to continue the conversation in a separate channel.
+
+Unwanted behavior
```
Comment (geekgonecrazy) [inline legal/code-of-conduct.md]: Should this be a header?  So we can draw attention to this section as well as link to it if needed?

File: legal/code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
```
Comment (geekgonecrazy) [inline legal/code-of-conduct.md]:
```suggestion
### Unwanted behavior
```

Maybe?

File: _layouts/en/legal.html [hunk 5]
```diff
@@ -11,6 +11,7 @@
           <li class="{% if page.type == 'privacy' %}double-column__links-current{% endif %}"><a href="/privacy">Privacy</a></li>
           <li class="{% if page.type == 'dmca' %}double-column__links-current{% endif %}"><a href="/dmca">DMCA Policy</a></li>
           <li class="{% if page.type == 'gdpr' %}double-column__links-current{% endif %}"><a href="/gdpr">GDPR</a></li>
+          <li class="{% if page.type == 'gdpr' %}double-column__links-current{% endif %}"><a href="/code-of-conduct">Code of Conduct</a></li>
```
Comment (guicruzzs) [inline _layouts/en/legal.html]: i guess the page.type should be `code-of-conduct` instead `gdpr` to apply `double-column__links-current` class

Comment (guicruzzs) [inline _layouts/en/legal.html]:
```suggestion
          <li class="{% if page.type == 'code-of-conduct' %}double-column__links-current{% endif %}"><a href="/code-of-conduct">Code of Conduct</a></li>
```

**Pull request merged**